### PR TITLE
changes for 1.1

### PR DESCRIPTION
**Pull request by abuehrle**
An initial edit @errordeveloper. We need to update the Apache image for this though, but I've edited this as is and also swapped out 1.01 for 1.02.

### CODE REVIEW

File: ubuntu-simple/DockerfileREADME.md [hunk 1]
```diff
@@ -1,9 +1,11 @@
 # Getting started with Weave and Docker on Ubuntu - Dockerfile #
 
-For our getting started with Weave and Docker on Ubuntu guide we created a small docker image, `fintanr/weave-gs-simple-hw`.  We have included the Dockerfile we used for creating this Docker image in our repo. While this is a very simple example it demonstrates how easy it is to create Docker images.
+For this _Getting Started with Weave and Docker on Ubuntu Guide_ we created a small docker image, `fintanr/weave-gs-simple-hw` and also included the Dockerfile that was used to create this Docker image. 
+
+While this is a very simple example, it demonstrates how easy it is to create Docker images.
 
 ```bash
-MAINTAINER    fintan@weave.works
+MAINTAINER    ilya@weave.works
```
Comment (errordeveloper) [inline ubuntu-simple/DockerfileREADME.md]: I would just take this line out completely or specify `help@weave.works` instead.

File: ubuntu-simple/README.md [hunk 2]
```diff
@@ -6,166 +6,208 @@ keywords: weave, docker, containers, networking, software defined networking, do
...
+sudo weave launch-dns
+~~~
+
+Your two hosts are now connected to each other, and any subsequent containers you launch with Weave will be visible to any other containers that the Weave network is aware of.
 
-Your two hosts are now connected to each other, and any subsequent containers you launch with Weave will be visible to other containers Weave is aware of.
+### What Just Happened? ###
 
-### What has happened? ###
+Since this is the first time launched Weave you:
```
Comment (errordeveloper) [inline ubuntu-simple/README.md]: Seems there another "you" missing here?

File: ubuntu-simple/README.md [hunk 3]
```diff
@@ -6,166 +6,208 @@ keywords: weave, docker, containers, networking, software defined networking, do
...
+
+## Deploying the Hello World Service ##
+
+Next, use Weave to run a Docker image containing an Apache webserver.  Details on how this container was created using docker are available [here](https://github.com/weaveworks/guides/blob/master/ubuntu-simple/DockerfileREADME.md).
 
 On `weave-gs-01` run
 
-    sudo weave run --name=web1 -t -i fintanr/weave-gs-simple-hw
+~~~bash
+sudo weave run --name=web1 -t -i fintanr/weave-gs-simple-hw
```
Comment (errordeveloper) [inline ubuntu-simple/README.md]: This should use `docker run` instead, for which you will need to do `eval $(weave proxy-env)` first...

File: ubuntu-simple/README.md [hunk 4]
```diff
@@ -175,18 +217,25 @@ with the addition of curl.
 First attach to your Docker container using the `CONTAINER` value we captured earlier
 
     sudo docker attach $CONTAINER
```
Comment (errordeveloper) [inline ubuntu-simple/README.md]: And here you don't need `docker attach` anymore, just user `docker run -ti ...`.

File: ubuntu-simple/README.md (hunk 3 above)
Comment (abuehrle) [inline ubuntu-simple/README.md]: I've ssh'ed on to the machine. Should we be using the proxy here as well?

**Comment by errordeveloper** [comment]
+1

**Pull request merged**